Sources/Controls/UIScrollView+Combine.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIScrollView+Combine.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 09/08/2019.
+//  Copyright © 2019 Shai Mishali. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+public extension UIScrollView {
+    /// A publisher emitting content offset changes from this UIScrollView.
```

Comment:
still 4 spaces
I'm thinking of moving everything to 4 anyways so I think I'll leave with it.